api/src/config.js

```diff
@@ -133,32 +134,33 @@ module.exports = {
     return loadSettings();
   },
   listen: () => {
-    db.medic.changes({ live: true, since: 'now' })
+    db.medic
+      .changes({ live: true, since: 'now' })
       .on('change', change => {
         if (change.id === '_design/medic') {
-          console.log('Detected ddoc change - reloading');
+          logger.info('Detected ddoc change - reloading');
           translations.run().catch(err => {
-            console.error('Failed to update translation docs', err);
+            logger.error(`Failed to update translation docs: ${err}`);
```

Comment:
I'm a little concerned about losing the stacktrace here. We might need to look at a solution like this: https://github.com/winstonjs/winston/issues/1243#issuecomment-376327966

Play around with throwing errors and see what gets logged.